src/lib/lib.jl

```diff
@@ -30,13 +30,13 @@ end
 
 @adjoint deepcopy(x) = deepcopy(x), ȳ -> (ȳ,)
 
-@adjoint (::Type{V})(x...) where V<:Val = V(x...), _ -> nothing
+@adjoint (::Type{V})(x...) where V<:Val = V(x...), _ -> DoesNotExist()
```

Comment:
Probably we want to undo all these and just make the change in ZygoteRules to handle the conversion?